lib/Tool/CandidateMapUtils.cpp

```diff
@@ -43,6 +72,18 @@ void souper::AddModuleToCandidateMap(InstContext &IC, ExprBuilderContext &EBC,
   }
 }
 
+static bool isInferDFA() {
+  return InferNeg || InferNonNeg || InferKnownBits || InferPowerTwo ||
+         InferNonZero || InferSignBits || InferRange;
+}
+
+static std::string convertToStr(bool Fact) {
```

Comment:
there are already two functions in Souper that do this, do we really need a third?

Reply:
Oops I’ll get it fixed